contracts/connectors/loantoken/modules/LoanTokenLogicWrbtc.sol

```diff
@@ -22,7 +22,7 @@ contract LoanTokenLogicWrbtc is LoanTokenLogicStandard {
 	 * @return The list of function signatures (bytes4[])
 	 */
 	function getListFunctionSignatures() external pure returns (bytes4[] memory functionSignatures, bytes32 moduleName) {
-		bytes4[] memory res = new bytes4[](35);
+		bytes4[] memory res = new bytes4[](36);
```

Comment:
`withdrawRBTCTo` probably should never be called for wrbtc loans. If it is true, consider moving the function into LoanTokenLogicLM.